requests/structures.py

```diff
@@ -90,7 +90,7 @@ class LookupDict(dict):
 
     def __init__(self, name=None):
         self.name = name
-        super(LookupDict, self).__init__()
+        super(self.__class__, self).__init__()
```

Comment:
This is not correct. The class name must always be hard-coded when using ``super()``.